pkgs/top-level/all-packages.nix

```diff
@@ -21650,7 +21650,7 @@ with pkgs;
 
   wcslib = callPackage ../development/libraries/science/astronomy/wcslib { };
 
-  webkitgtk = callPackage ../development/libraries/webkitgtk {
+  webkitgtk = darwin.apple_sdk_11_0.callPackage ../development/libraries/webkitgtk {
```

Comment:
This deserves an explanation comment in the code.

Reply:
Done.